Parse/build.gradle

```diff
@@ -36,14 +38,22 @@ android {
             testCoverageEnabled = true
         }
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_7
+        targetCompatibility JavaVersion.VERSION_1_7
+    }
+
+
+
+    useLibrary 'org.apache.http.legacy'
 }
 
 dependencies {
+    compile 'com.android.tools.build:gradle:2.0.0-beta6'
     compile 'com.parse.bolts:bolts-tasks:1.4.0'
-
+    compile 'org.apache.httpcomponents:httpclient-android:4.3.5.1'
```

Comment:
This is unnecessary since you already have `useLibrary 'org.apache.http.legacy'`